src/sparseml/pytorch/utils/__init__.py

```diff
@@ -18,13 +18,13 @@
 
 # flake8: noqa
 
+from ...core.logger import *
```

Comment:
instead of importing from `.core` here, let's still keep `src/sparseml/utils/logger.py` and have it import from core.loggers and add the relevant classes to the `__all__` list, and then add a note that it's been moved

Reply:
Done!